ui/src/createAzureClient.ts

```diff
@@ -29,7 +29,7 @@ export function createAzureSubscriptionClient<T>(
     });
 }
 
-export function createGenericClient(clientInfo?: ServiceClientCredentials | { credentials: ServiceClientCredentials; environment: Environment; }): ServiceClient {
+export async function createGenericClient(clientInfo?: ServiceClientCredentials | { credentials: ServiceClientCredentials; environment: Environment; }): Promise<ServiceClient> {
```

Comment:
The other methods in this file could also be made `async` for consistency--though I haven't inspected all of their consumers--but I opted to err on the side of less breaking changes instead. I'm OK with either.